src/Compilers/CSharp/Portable/Symbols/SignatureOnlyParameterSymbol.cs

```diff
@@ -90,7 +90,7 @@ internal override ScopedKind EffectiveScope
 
         internal override ImmutableHashSet<string> NotNullIfParameterNotNull { get { throw ExceptionUtilities.Unreachable(); } }
 
-        public override Symbol ContainingSymbol { get { throw ExceptionUtilities.Unreachable(); } }
+        public override Symbol? ContainingSymbol => null;
```

Comment:
Why is this change needed? Also shouldn't this point to some actual containing symbol rather than `null`?

Reply:
Thanks. It's no longer needed (was left over from earlier experiment with this)